docs/content/references/cli/ptb.mdx

```diff
@@ -108,11 +109,12 @@ When needing to execute a move call, use the `--move-call` transaction to call a
 Publishing a package is one of the most important commands that users need when working with Sui. While the CLI has a standalone `publish` command, PTBs also supports publishing and upgrading packages. One main difference is that with `sui client ptb`, you would need to explicitly transfer the upgrade cap object that is returned when creating a package, or destroy it with a call to [`make_immutable`](/concepts/sui-move-concepts/packages.mdx). Here is an example on how to publish a Move project on chain using the `sui client ptb` command. It makes a call to the `sui::tx_context::sender` to acquire the sender and assigns the result of that call to the `sender` variable, and then calls the publish command. The result of `publish` is bounded to `upgrade_cap` variable, and then this object is transferred to the sender. 
 
 ```bash
---move-call sui::tx_context::sender
---assign sender
---publish "."
---assign upgrade_cap
---transfer-objects sender "[upgrade_cap]"
+sui client ptb \
+--move-call sui::tx_context::sender \
+--assign sender \
+--publish "." \
+--assign upgrade_cap \
+--transfer-objects [upgrade_cap] sender \
```

Comment:
nit: this change also removes the quotes around `"[upgrade_cap]"`, is that intentional (i.e. will this PTB run without quotes in bash?)

Reply:
Yes, bash does not require quotes around `[upgrade_cap]`, only around types `<u64>`.